ocsmesh/hfun/raster.py

```diff
@@ -31,6 +33,14 @@
 _logger = logging.getLogger(__name__)
 
 
+def _apply_constraints(method):
```

Comment:
This is just a shorthand decorator to easily add to methods that require re-application of constraints.